databricks/koala/frame.py

```diff
@@ -59,12 +58,10 @@ def _index_columns(self):
         return [anchor_wrap(self, self._spark_getitem(field))
                 for field in self._metadata.index_fields]
 
-    @derived_from(pd.DataFrame)
```

Comment:
Let's keep them, because they add documentation from the pandas Dataframe automatically. I just tried and here is what it looks like:

![Screenshot 2019-04-18 at 10 52 25](https://user-images.githubusercontent.com/7594753/56349061-1a0e2980-61c8-11e9-93de-58dd7d58a8cb.png)

Reply:
Hm, I removed this because the doctests were wrongly copied too. But seems they are supposed to be skipped. Let me take a look.

Reply:
Hm, let me handle this separately.